packages/gatsby-telemetry/src/index.js

```diff
@@ -7,7 +7,10 @@ process.on(`exit`, flush)
 
 // For longrunning commands we want to occasinally flush the data
 // The data is also sent on exit.
-const interval = 10 * 60 * 1000 // 10 min
+
+const interval = process.env.TELEMETRY_BUFFER_INTERVAL
```

Comment:
Could you add some checks to ensure that interval is always a number? If it ends up as `NaN` or any falsy value then the timeouts will fire immediately.

Reply:
Oh just saw @pieh's comment :) I think this could still cause problems depending on the input. e.g: `Math.max(Number('oops this is a string'), 1000)` will be `NaN`.

Reply:
Indeed

Reply:
Updated, is the Number.isFinite too much of a hack?

Reply:
Nah that should be good, thanks!

Reply:
Available in Node 0.10 :) Not _that_ hacky ™️